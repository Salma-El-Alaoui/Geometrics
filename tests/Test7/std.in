C c1 1 1 1
PL pl1 1 1 1 1 1 1 1 1
LIST
DELETE c1 pl1
LIST
C c1 1 1 1
R r1 1 1 1 1
L l1 1 1 1 1
PL pl1 1 1 1 1 1 1
LIST
DELETE r1 l1
LIST
DELETE c1 pl1
LIST
EXIT
