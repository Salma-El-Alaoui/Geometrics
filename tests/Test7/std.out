OK
#Element ajoute: c1
OK
#Element ajoute: pl1
C c1 1 1 1
PL pl1 1 1 1 1 1 1 1 1
#Element c1 supprime
#Element pl1 supprime
OK
OK
#Element ajoute: c1
OK
#Element ajoute: r1
OK
#Element ajoute: l1
OK
#Element ajoute: pl1
C c1 1 1 1
L l1 1 1 1 1
PL pl1 1 1 1 1 1 1
R r1 1 1 1 1
#Element r1 supprime
#Element l1 supprime
OK
C c1 1 1 1
PL pl1 1 1 1 1 1 1
#Element c1 supprime
#Element pl1 supprime
OK
